README.md

```diff
@@ -66,15 +66,21 @@ Don't forget to add this configuration to the main `.nextflow.config`.
 
 ## Data download
 
-If you use EBI cluster:
+If you use ENA data:
 1) Get your Raw reads and Assembly study accessions;
-2) Download data from ENA, [get assembly and run_accessions](download_data/scripts/rename-erz.py) and [generate](download_data/scripts/generate_samplesheet.py) input samplesheet:
+2) Download data: get assemblies and runs from ENA and [generate](download_data/scripts/generate_samplesheet.py) input samplesheet.csv and software.tsv; 
+
+Additional filters:
+- filter out METATRANSCRIPTOMIC runs specify `-t` (GGP currently does not support metaT)
+- filter by scientific_name use `-b SCIENTIFIC_NAME` (like in ENA records)
 ```commandline
 bash download_data/fetch_data.sh \
     -a assembly_study_accession \
     -r reads_study_accession \
     -c `pwd`/assembly_study_accession \
-    -f "false"
+    -f "false" \
+    -b "marine sediment metagenome" \
```

Comment:
We should allow a list here. I think comma-separated would work since you are basically carrying this over to a python script.